### ISSUE TITLE
Hope To Support Chinese！

### ISSUE THREAD

**Issue by kingrake** [SwiftForWindows/SwiftForWindows#64]
Hope To Support Chinese！希望支持中文！！！！！一开就闪退！或者是乱码！

**Comment by tinysun212** [comment]
Could you please show me the screenshot that caused the problem?

**Comment by kingrake** [comment]
Actually,about é can't type correct! e.x. let cofee = "cof\u{0065}\u{0301}"!

**Comment by kingrake** [comment]
I think you use wrong Unicode!

**Comment by fansxs** [comment]
yes, this software is not use utf8 encoding, hope change to utf8 encoding to compile. :)

**Comment by fansxs** [comment]
Like javac -encoding utf-8  TestInteger.java
